src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -0,0 +1,372 @@
+            return;
+        }
+
+        bool result = Directory.Exists(Path.Combine(drive, "Subdirectory"));
+
+        Assert.False(result);
+    }
+
+    [Fact]
+    public static void Exists_NonExistentDriveAsPath_ReturnsFalse()
+    {
+        var drive = IOServices.GetNonExistentDrive();
+        if (drive == null)
+        {
+            Console.WriteLine("Skipping test. Unable to find a non-existent drive.");
+            return;
+        }
+
+        bool result = Directory.Exists(drive);
+
+        Assert.False(result);
+    }
+
+    [Fact]
+    public static void Exists_SubdirectoryOnNonExistentDriveAsPath_ReturnsFalse()
+    {
+        var drive = IOServices.GetNonExistentDrive();
+        if (drive == null)
+        {
+            Console.WriteLine("Skipping test. Unable to find a non-existent drive.");
+            return;
+        }
+
+        bool result = Directory.Exists(Path.Combine(drive, "Subdirectory"));
+
+        Assert.False(result);
+    }
+
+
+    [Fact]
+    public static void Exists_FileWithoutTrailingSlashAsPath_ReturnsFalse()
+    {
+        using (TemporaryFile file = new TemporaryFile())
+        {
+            string path = IOServices.RemoveTrailingSlash(file.Path);
+
+            bool result = Directory.Exists(path);
+
+            Assert.False(result);
+        }
+    }
+
+    [Fact]
+    public static void Exists_FileWithTrailingSlashAsPath_ReturnsFalse()
+    {
+        using (TemporaryFile file = new TemporaryFile())
+        {
+            string path = IOServices.AddTrailingSlashIfNeeded(file.Path);
+
+            bool result = Directory.Exists(path);
+
+            Assert.False(result);
+        }
+    }
+
+    [Fact]
+    public static void Exists_ExistingDirectoryWithoutTrailingSlashAsPath_ReturnsTrue()
+    {
+        using (TemporaryDirectory directory = new TemporaryDirectory())
+        {
+            string path = IOServices.RemoveTrailingSlash(directory.Path);
+
+            bool result = Directory.Exists(path);
+
+            Assert.True(result);
+        }
+    }
+
+    [Fact]
+    public static void Exists_ExistingDirectoryWithTrailingSlashAsPath_ReturnsTrue()
+    {
+        using (TemporaryDirectory directory = new TemporaryDirectory())
+        {
+            string path = IOServices.AddTrailingSlashIfNeeded(directory.Path);
+
+            bool result = Directory.Exists(path);
+
+            Assert.True(result);
+        }
+    }
+
+    [Fact]
+    public static void Exists_DotAsPath_ReturnsTrue()
+    {
+        bool result = Directory.Exists(TestInfo.CurrentDirectory + @"\.");
+
+        Assert.True(result);
+    }
+
+    [Fact]
+    public static void Exists_DotDotAsPath_ReturnsTrue()
+    {
+        bool result = Directory.Exists(TestInfo.CurrentDirectory + @"\..");
+
+        Assert.True(result);
+    }
+
+#if !TEST_WINRT // WinRT cannot access root
+    /*
+    [Fact]
+    [ActiveIssue(0)] // SetCurrentDirectory
```

Comment:
Need something here other than just ActiveIssue(0).  Either file an issue or delete the test.